.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,99 @@
+name: Bug report
+description: Create a bug report to help us improve
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for making NewPipe better by reporting a bug. :hugs:
+
+        Please fill in as much information as possible about your bug so that we don't have to play "information ping-pong" but can help you immediately.
+
+  - type: checkboxes
+    id: checklist
+    attributes:
+      label: "Checklist"
+      options:
+        - label: "I am able to reproduce the bug with the [latest version](https://github.com/TeamNewPipe/NewPipe/releases/latest)."
+          required: true
+        - label: "I made sure that there are *no existing issues*, neither [open](https://github.com/TeamNewPipe/NewPipe/issues) nor [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed), which I could contribute my information to."
+          required: true
+        - label: "I am going to take the time to to fill in all the required details. I know that the bug report will be dismissed otherwise."
+          required: true
+        - label: "This issue contains only one bug. I will open one issue for every bug report I want to file."
```

Comment:
```suggestion
        - label: "This issue contains only one bug report. I will open one new issue for every bug report I want to file."
```

Reply:
> I will open one issue for every bug report I want to file.

This actually has the opposite meaning. As in "I will open only one issue and put all my bug reports in it."

Reply:
Removed the "I will open one ..." - sentence.
I think a user normally should understand through the first sentence that he/she should create another issue for additional bugs/features.